orbital-mechanics/Lecture7/Lecture7.ipynb

```diff
@@ -1,386 +1,425 @@
 {
```

Comment:
<p>Equations in their own line again, please.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angadhn/SpacecraftDynamics/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>For the equations in 1., should I just give them there own line or a equation number as well like the equations in 2?</p>

Reply:
<p>Will it look odd if the " and" in 1. and 4. is aligned to the left and the equation is in the centre. I found a way of bringing the and in the centre:</p>

<pre class="ql-syntax" spellcheck="false">$$
\text{and}
$$
</pre><p>It appears more text like in VSCode but I can still tell it is different from main body text. I'm not sure how it'll look on the website.</p>

Reply:
<p>That's fine.</p>